pkg/operator/mon/mon.go

```diff
@@ -411,11 +411,34 @@ func (c *Cluster) saveMonConfig() error {
 
 // detect the nodes that are available for new mons to start.
 func (c *Cluster) getAvailableMonNodes() ([]v1.Node, error) {
+	availableNodes, nodes, err := c.getMonNodes()
+	if err != nil {
+		return nil, err
+	}
+	logger.Infof("Found %d running nodes without mons", len(availableNodes))
+
+	// if all nodes already have mons, just add all nodes to be available
+	if len(availableNodes) == 0 {
+		logger.Infof("All nodes are running mons. Adding all %d nodes to the availability.", len(nodes.Items))
+		for _, node := range nodes.Items {
+			if validNode(node, c.placement) {
+				availableNodes = append(availableNodes, node)
+			}
+		}
+	}
+	if len(availableNodes) == 0 {
+		return nil, fmt.Errorf("no nodes are available for mons")
+	}
+
+	return availableNodes, nil
+}
+
+func (c *Cluster) getMonNodes() ([]v1.Node, *v1.NodeList, error) {
```

Comment:
can you switch the names of these methods? This one seems better as `getAvailableMonNodes` and the other one `getMonNodes`